app/uk/gov/hmrc/hmrcemailrenderer/templates/tdq/tdqFphReportHeuristicallyCompliant.scala.html

```diff
@@ -0,0 +1,50 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.hmrcemailrenderer.templates.tdq.params.TdqFphReportParams
+@(params: Map[String, Any])
+@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params, "Fraud prevention headers report") {
+    @defining(TdqFphReportParams(params)) { reportParams =>
+        <h2>VAT Making Tax Digital (MTD)</h2>
+
+        <p style="font-size: 19px; font-weight: bold; line-height: 1.315789474; margin: 0 0 30px 0;">@reportParams.applicationName</p>
+
+        <p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">Dear @reportParams.developerName</p>
+
+        <p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">
+            To help protect your customers' confidential data from criminals and fraudsters, we monitor API calls and send you feedback every month.
+        </p>
+
+        <p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">
+            Our logs show that all of the fraud prevention headers submitted for @reportParams.applicationName between
```

Comment:
Do we want to put quotes around application name?

Reply:
Mia's template and example don't use quotes around the application name